contributions/compare.ts

```diff
@@ -0,0 +1,24 @@
+import { Targeting } from "../types";
+
+interface CompareMeta {
+  targeting: Targeting;
+  expectedTest: string;
+  expectedVariant: string;
+}
+
+const apiURL =
+  "https://contributions.guardianapis.com/epic/compare-variant-decision";
```

Comment:
I think this is good as-is. If we keep it around for longer then maybe we should think about combining the various calls we make to the service in an api module? That would allow us to share things like the base URL config and any other common concerns around making HTTP requests.